src/test/java/e2e/config/ConfigurationProvider.java

```diff
@@ -0,0 +1,42 @@
+package e2e.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.io.Resources;
+import com.google.inject.Singleton;
+import io.dropwizard.configuration.ConfigurationException;
+import io.dropwizard.configuration.ConfigurationFactory;
+import io.dropwizard.jackson.Jackson;
+import io.dropwizard.validation.BaseValidator;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+@Singleton
+public class ConfigurationProvider {
+
+    private static final ObjectMapper objectMapper = Jackson.newObjectMapper();
+    private static final ConfigurationFactory<TestConfiguration> factory = new ConfigurationFactory<>(
+            TestConfiguration.class,
+            BaseValidator.newValidator(),
+            objectMapper, "dw");
+
+    private static TestConfiguration config;
+
+    private static TestConfiguration parseFile() {
+        try {
+            URI path = Resources.getResource("test_config.yml").toURI();
+            return factory.build(new File(path));
+        } catch (IOException | URISyntaxException | ConfigurationException e) {
+            return null;
```

Comment:
This will cause NullPointerExceptions without preserving the original stacktrace - I think it's better to re-throw an exception instead of catching it and returning nothing